api/src/main/java/io/grpc/SynchronizationContext.java

```diff
@@ -149,16 +151,16 @@ public final ScheduledHandle schedule(
       final Runnable task, long delay, TimeUnit unit, ScheduledExecutorService timerService) {
     final ManagedRunnable runnable = new ManagedRunnable(task);
     ScheduledFuture<?> future = timerService.schedule(new Runnable() {
-        @Override
```

Comment:
Don't format lines unrelated to your changes, it will make git blame difficult to identify changes specific to a fix.

Reply:
The formatting changes have not been reverted, here and in other places.